### DISCUSSION TITLE
Do I need to do anything with the IMPORT

### DISCUSSION THREAD

**Discussion by bonelifer** [RandomNinjaAtk/arr-scripts#249] [Q&A] [answered]
Do I need to do anything with the IMPORT directory, or does it periodically get dumped in the right place?

**Comment by RandomNinjaAtk** [comment]
If you have queuecleaner enabled, you shouldn’t have to do anything. If you don’t have it enabled, you will likely need to manually clear the failed imports periodically.

**Comment by bonelifer** [comment]
from extended.conf
```
enableQueueCleaner="true" 
```

Files start out in incomplete, I assume beets does it stuff,  then when that's done, they appear in imports and stay there, not moved to the root, I started with a new install, where  I created a root called CURRENT, which is a folder in downloads, that contain, my current library.  I imported it, set to monitor all. My current collection is 50,000'ish tracks. If I move those files, it will re-download them. Needless to say it has its work cut out after following thousands of Artists.  Is there something in Lidarr, I may of changed that is effecting this?

**Reply by RandomNinjaAtk** [reply]
Make sure the download client "Arr-Extended" is in the download clients page of lidarr web interface and pointed at the correct import folder (same one specified in the conf file). FYI: If you changed it from the default location, its very possible its not pointed to the correct location because the script cannot update the client when changes are made to the path in the conf file.

**Comment by bonelifer** [comment]
It is not in the Setting --> Download Clients page. I didn't delete it either. I didn't change that part due to it saying not to in extended.conf.

```
/homelab/lidarr/data/lidarr/config/extended/import/
```

**Reply by RandomNinjaAtk** [reply]
If it’s not there, your either not on the latest version of script or it hasn’t  downloaded anything lately, because the client is only added right before executing a download.

**Comment by bonelifer** [comment]
content of `/homelab/lidarr/data/custom-cont-init.d/scripts_init.bash`, I assume I leave that there after install.,
```
#!/usr/bin/with-contenv bash
curl https://raw.githubusercontent.com/RandomNinjaAtk/arr-scripts/main/lidarr/setup.bash | bash
exit
```

These are the only thing I changed in `extended.conf`:
```
arlToken=
audioBitrate="low"
requireQuality="true" 
```

**Reply by RandomNinjaAtk** [reply]
That’s fine, but as I said the client is only added to lidarr when something is downloaded. Until then, you won’t have it listed in the web interface.

**Comment by bonelifer** [comment]
I never see it in client, and it's still downloading.

**Comment by bonelifer** [comment]
Its downloading right now and nothing in clients page.

**Reply by RandomNinjaAtk** [reply]
Post logs from the script to show the full download process. Without logs nothing more can be done

**Reply by bonelifer** [reply]
`/homelab/lidarr/data/lidarr/config/extended/logs`, all of those are empty?

**Reply by RandomNinjaAtk** [reply]
Logs are in /config/logs

**Comment by bonelifer** [comment]
[logs.tar.gz](https://github.com/RandomNinjaAtk/arr-scripts/files/15155363/logs.tar.gz)

**Reply by RandomNinjaAtk** [reply]
Thanks, I just realized what the problem is. It appears when I fixed a bug to prevent errors in Lidarr, I also prevented it from adding the client if any client exists most likely....

I'll hopefully post a fix soon and let you know when to give it a try...

**Comment by RandomNinjaAtk** [comment, marked as answer]
This should be resolved with the latest change, please verify...

https://github.com/RandomNinjaAtk/arr-scripts/commit/7c4ee17274769f558afc78e5871be2b1252bdf6a

**Comment by bonelifer** [comment]
Still not showing th ARR-SCRIPTS client and just leaving the downloads in imports instead of moving to the CURRENT folder.

![lidarr](https://github.com/RandomNinjaAtk/arr-scripts/assets/121736/b01559bd-b4b4-4487-bea6-43648b997c2a)
[logs.tar.gz](https://github.com/RandomNinjaAtk/arr-scripts/files/15156694/logs.tar.gz)

**Reply by RandomNinjaAtk** [reply]
Your logs show you were running v2.43 and not the latest 2.44. So update....
